### PR TITLE
Shell script: Fix condition in make.sh script

### PR DESCRIPTION
**Pull request by iLLiCiTiT**
## Changelog Description
The condition caused `[: -eq: unary operator expected` which should be fixed now.

## Testing notes
1. Make sure `.poetry` is not in root of repository.
2. Run `./tools/make.sh create-env`.
3. Script should not complain about invalid syntax.

### CODE REVIEW

**Review by Lypsolon — COMMENTED** [review]
Testing notes would be great.

**Comment by iLLiCiTiT** [comment]
> Testing notes would be great.

I though this change does not really need any testing?

**Comment by Lypsolon** [comment]
> > Testing notes would be great.
> 
> I though this change does not really need any testing?

Well, you put me for code review, and I will admit I don't know how and when the if test would be true. So, to check if it's actually doing the expected behaviour, I would need testing notes. 

if you just want me to look at the code, yes, it looks correct, but that's not a review.

**Comment by iLLiCiTiT** [comment]
> if you just want me to look at the code, yes, it looks correct, but that's not a review.

It is review. Code review does not always require testing.

File: tools/make.sh [hunk 1]
```diff
@@ -146,7 +146,7 @@ install_poetry () {
   ssl_command="import ssl;print(1 if ssl.OPENSSL_VERSION_INFO < (1, 1, 1) else 0)"
   local downgrade_urllib
   downgrade_urllib="$("$poetry_home_root/venv/bin/python" <<< ${ssl_command})"
-  if [ $downgrade_urllib -eq "1" ]; then
+  if [[ $downgrade_urllib -eq "1" ]]; then
```
Comment (antirotor) [inline tools/make.sh:149]:
```suggestion
  if [ "$downgrade_urllib" -eq "1" ]; then
```
maybe better fix is this. So it is more portable as it isn't using bash extensions.

**Pull request merged**